bundles/io/org.eclipse.smarthome.io.rest.core/META-INF/MANIFEST.MF

```diff
@@ -5,18 +5,23 @@ Bundle-SymbolicName: org.eclipse.smarthome.io.rest.core
 Bundle-Version: 0.9.0.qualifier
 Bundle-Vendor: Eclipse.org
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
-Import-Package: com.google.common.base,
+Import-Package: com.eclipsesource.jaxrs.provider.security,
```

Comment:
I'd suggest to not make everbody dependent on the jax-rs connector security bundle.
We should move the `SmartHomePrincipal` and the `AbstractSecurityHandler` to a new bundle o.e.s.io.rest.auth and the `BasicSecurityHandler` implementation to a new bundle o.e.s.io.rest.auth.basic.